app/src/ui/diff/side-by-side-diff-row.tsx

```diff
@@ -14,6 +14,7 @@ import { narrowNoNewlineSymbol } from './text-diff'
 import { shallowEquals, structuralEquals } from '../../lib/equality'
 import { DiffHunkExpansionType } from '../../models/diff'
 import { DiffExpansionKind } from './text-diff-expansion'
+import { String } from 'aws-sdk/clients/xray'
```

Comment:
Looks like you are just passing in a regular string - no need to include `String` from this package.